docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,329 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Feature and Component Statuses
+==============================
+
+This page gives an overview of the statuses of released components and features according to :doc:`status-definitions`. Anything not listed here implicitly has status "Labs", but it's possible that something accidentally slipped the list so if in doubt, please :doc:`contact us <support>`.
+
+Ledger API
+----------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component/Feature
+     - Status
+     - Deprecated on
+   * - :doc:`Ledger API specification </app-dev/grpc/proto-docs>` including all semantics of >= DAML-LF 1.6
+     - Stable
+     -   
+   * - CLI and test names of :doc:`/tools/ledger-api-test-tool/index`
+     - Beta
+     -
+   * - `Numbered (ie non-dev) Versions of Proto definitions distributed via GitHub Releases <protobufs_>`_
+     - Stable
+     - 
+   * - `Dev Versions of Proto definitions distributed via GitHub Releases <protobufs_>`_
```

Comment:
It’s worth pointing out that we do not have a dev version of anything but DAML-LF. For example, `GenMap` is already part of the stable value protobuf but only in DAML-LF dev.

Reply:
Why do you think that needs pointing out here?

Reply:
Sorry I didn’t mean to point it out to the docs but point it out to ourselves. This means that LF dev doesn’t mean we can remove things since they are already in the stable value definition. Or we deliberately ignore that until we have a dev-version of values.

Reply:
@remyhaemmerle-da: could you point us the section in the LF spec where value/transaction versioning is specified please?

Reply:
+ values are specified in `daml-lf/spec/value.rst`
+ transactions are specified in `daml-lf/spec/transaction.rst`